### ISSUE TITLE
Cannot change the language by default

### ISSUE THREAD

**Issue by danimondurrey** [PowerBiDevCamp/TranslationsBuilder#16]
We cannot change the language by default in Translations Builder. We have tried from Power BI Desktop changing regional settings and it is not working.
If you know a workaround, please let us know
Thanks!

**Comment by TeemuVSaarni** [comment]
I don't think this can or even should be handled by translation editor at all. It should be a user based setting as it is now, whether in power bi settings or in browser language setting.
